Packs/CortexXDR/IncidentFields/XDR_Host_Count.json

```diff
@@ -0,0 +1,39 @@
+{
+	"associatedToAll": false,
+	"associatedTypes": [
+		"Cortex XDR Incident"
+	],
+	"breachScript": "",
+	"caseInsensitive": true,
+	"cliName": "xdrhostcount",
+	"closeForm": false,
+	"columns": null,
+	"content": false,
+	"defaultRows": null,
+	"description": "",
+	"editForm": true,
+	"fieldCalcScript": "",
+	"group": 0,
+	"hidden": false,
+	"id": "incident_xdrhostcount",
+	"isReadOnly": false,
+	"locked": false,
+	"name": "XDR Host Count",
+	"neverSetAsRequired": false,
+	"ownerOnly": false,
+	"placeholder": "",
+	"required": false,
+	"script": "",
+	"selectValues": null,
+	"sla": 0,
+	"system": false,
+	"systemAssociatedTypes": null,
+	"threshold": 72,
+	"type": "number",
+	"unmapped": false,
+	"unsearchable": false,
+	"useAsKpi": false,
+	"validationRegex": "",
+	"version": -1,
+	"fromVersion": "5.0.0"
```

Comment:
@bakatzir you marked in the PR minimum version 4.1.0
probably a mistake in the PR

Reply:
True, both classifier and incident field changes are relevant for 5.0 onwards.